CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 1.4.0
+* Feature - Add support for dynamic log group names (#46)
+* Feature - Add support for dynamic log stream names (#16)
+* Feature - Support tagging of newly created log groups (#51)
+* Feature - Support setting log group retention policies (#50)
+
 ## 1.3.1
```

Comment:
Just wondering, we didn't add any changelog for v1.3.2?

Reply:
I think since there were no customer facing changes, we didn't: https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit/commit/d3ce6925ec02bf65865573f47cfea96ca20cc78c

that was back when we released directly from mainline. After it yenlin changed the Dockefile to pull from a tag instead